doc/contribute/build_ccf.rst

```diff
@@ -9,24 +9,17 @@ First, checkout the CCF repository:
 
     $ git clone git@github.com:microsoft/CCF.git
 
-To build CCF from source on a SGX-enabled machine, run the following:
-
-.. code-block:: bash
-
-    $ cd CCF
-    $ mkdir build
-    $ cd build
-    $ cmake -GNinja ..
-    $ ninja
-
-Alternatively, on a non-SGX machine, you can build a `virtual` instance of CCF:
+To build CCF from source, run the following:
 
 .. code-block:: bash
 
     $ cd CCF
     $ mkdir build
     $ cd build
+    # Virtual build, insecure but can run anywhere
     $ cmake -GNinja -DCOMPILE_TARGET=virtual ..
+    # SEV-SNP build, secure but requires supported hardware to run
+    $ cmake -GNinja -DCOMPILE_TARGET=snp .. 
```

Comment:
I think displaying both of these in series is just going to confuse someone (ie - the people who don't know what cmake is, and are copy-pasting these commands). We should use platform-specific tabs, as we do elsewhere.

Reply:
Not directly related, but I would to like propose we drop our weird custom CMake build type default (RelWithDebInfo) and default to Debug like every other CMake project. I seem to remember that we did this because it was too easy to build and run the benchmarks with debug and get bad numbers, which feels like a lesser concern at this stage of the project's life than tooling consistency.

Reply:
I approve.